book/index.qmd

```diff
@@ -3,14 +3,6 @@ title: "TLG Catalog"
 subtitle: "Tables, Listings, and Graphs Catalog"
 ---
 
-<img src="./assets/img/tlgc.png" style="float: right;" width="200;"/>
+<img src="./assets/img/logo.png" style="float: right;" width="200;"/>
 
-## Introduction
-
-A catalog of Tables, Listings, and Graphs output produced by NEST tools.
-
-**See the full index of available TLG templates on the [Index page](tlg-index.qmd).**
-
-## License
-
-This catalog as well as code examples are licensed under the Apache License, Version 2.0.
+{{< include README.md >}}
```

Comment:
One thing that I still struggle with is the "Development" section that is present in the `README.md` and included there in the home page of the book. The informations outlined there is needed for _Developers_ and has to be somewhere (presumably README.md) but probably we don't want to expose it to the _Users_. I don't know how to best differentiate between _Users_ and _Developers_. I decided to keep it like this.

The alternative would be to copy & paste all sections except Development from README.md into index.qmd but I don't super like it as it's not DRY. 

Please let me know your thoughts on this. If you also see this as a problem and there are no good alternatives, we might apply not-DRY solution.

Reply:
Yes. That's also a valid option to consider. 
However, I have some doubts whether CONTRIBUTING would be actually looked into by the developers. Even though it's linked nicely in GH - I just don't observe signs of use in form of update requests or anything like that. I know this might sound a little contradictory to the fact that I included this file in this PR :)
I am looking for a place more used than CONTRIBUTING and less exposed to non-developer consumers of the book. 

Aghhh that's tough. I'm inclined to just go ahead with this as is. We can update at any time in the future. Also not sure if I am the only one who see it as an issue :)